### PR TITLE
[mesheryctl] added test for view and list

### LINKED ISSUE #3833: [mesheryctl] (child) `mesheryctl filter` unit testing
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

check Epic here #3016 

#### Current Behavior
Currently, we don't have any unit tests setup to test `mesheryctl filter` commands and functions

#### Desired Behavior
Required to enable unit testing support for `mesheryctl filter` commands and functions, to ensure the accuracy and robustness of each mesheryctl release.

**mesheryctl Unit Testing**
- Mark all possible situations for each `mesheryctl filter` commands and functions.
- Write tests using Golang’s standard library.
- A combination of CodeCov and GitHub Actions are to be used as mainstays in the approach to unit testing - https://github.com/codecov/codecov-action.

#### Subcommands to cover

- [ ] mesheryctl filter apply
- [ ] mesheryctl filter list
- [ ] mesheryctl filter delete
- [ ] mesheryctl filter view

---

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: Ashish Tiwari <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #3833 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/filter/view_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	}{
+		{
+			Name:             "Fetch Filter View",
+			Args:             []string{"view", "KumaTest"},
+			ExpectedResponse: "view.filter.output.golden",
+			Fixture:          "view.filter.api.response.golden",
+			URL:              testContext.BaseURL + "/api/experimental/filter",
+			Token:            filepath.Join(fixturesDir, "token.golden"),
+			ExpectError:      false,
+		},
```
Comment (piyushsingariya) [inline mesheryctl/internal/cli/root/filter/view_test.go:51]: we can add one for Id too.

Comment (Revolyssup) [reply mesheryctl/internal/cli/root/filter/view_test.go:52]: done

**Comment by Revolyssup** [comment]
@piyushsingariya  any more changes?

**Review by piyushsingariya — APPROVED** [review]
Yayy! more tests joining the club.

**Comment by piyushsingariya** [comment]
@Revolyssup let's hold on this PR, I'm thinking to change the output of the filter view.

**Pull request merged**